sr_robot_launch/scripts/sr_ur_arm_unlock

```diff
@@ -0,0 +1,166 @@
+        else:
+            service_string = "/" + side + "_sr_ur_robot_hw/" + service_name
+        try:
+            service_call = rospy.ServiceProxy(service_string, service_type)
+            response = service_call()
+            return response
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to '%s' failed for arm %s. %s", service_name, arm, e)
+
+    def startup_arms(self, arms):
+        for arm in arms:
+            self.call_arm_service(arm, "power_on", Trigger)
+        rospy.sleep(5)
+        for arm in arms:
+            self.call_arm_service(arm, "brake_release", Trigger)
+        rospy.sleep(5)
+
+    def check_arms_e_stops(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                while safety_mode.safety_mode.mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                    rospy.logwarn("Emergency stop button is still pressed for arm %s, please release", arm)
+                    safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+                    rospy.sleep(2)
+
+    def check_fix_arms_protective_stop(self, arms):
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.PROTECTIVE_STOP:
+                rospy.loginfo("Protective stop detected on: %s", arm)
+                self.call_arm_service(arm, "unlock_protective_stop", Trigger)
+
+    def check_fix_arms_fault(self, arms):
+        fault = False
+        for arm in arms:
+            safety_mode = self.call_arm_service(arm, "get_safety_mode", GetSafetyMode)
+            if safety_mode.safety_mode.mode == SafetyMode.FAULT:
+                fault = True
+                rospy.loginfo("Fault detected on: %s", arm)
+                rospy.loginfo("This can be caused by using the external e-stop. Have you released it?")
+                self.call_arm_service(arm, "restart_safety", Trigger)
+        return fault
+
+    def check_arms_robot_mode(self, arms):
+        arm_needs_starting = False
+        for arm in arms:
+            robot_mode = self.call_arm_service(arm, "get_robot_mode", GetRobotMode)
+            if robot_mode.robot_mode.mode == RobotMode.IDLE or robot_mode.robot_mode.mode == RobotMode.POWER_OFF:
+                rospy.loginfo("Starting arm: %s", arm)
+                arm_needs_starting = True
+        return arm_needs_starting
+
+    def clear_arms_popups(self, arms):
+        for arm in arms:
+            self.call_arm_service(arm, "close_safety_popup", Trigger)
+        rospy.sleep(2)
+        for arm in arms:
+            self.call_arm_service(arm, "close_popup", Trigger)
+        rospy.sleep(2)
+
+    def check_fix_program_loaded_arms(self, arms):
```

Comment:
its also loading, lets put that informationin the method name

Reply:
Yeah, again, I had some stupid method names :P 11c9f4658b1c2977eef546c8fbc1abe42126630c